apps/cowswap-frontend/src/modules/trade/hooks/useTradePriceImpactParams.ts

```diff
@@ -10,29 +10,15 @@ const emptyState = {
   outputCurrencyAmount: null,
 }
 
-export function useTradePriceImpactParams(): PriceImpactParams {
+export function useTradePriceImpactParams(): ParsedAmounts {
```

Comment:
This hook is now `useParsedAmounts`, no?

Reply:
Renamed, thanks!